core/src/main/scala/org/apache/spark/deploy/history/HistoryServerDiskManager.scala

```diff
@@ -238,10 +240,11 @@ private class HistoryServerDiskManager(
     new File(appStoreDir, fileName)
   }
 
-  private def updateAccessTime(appId: String, attemptId: Option[String]): Unit = {
+  private def updateApplicationStoreInfo(
```

Comment:
This function updates both the lastAccess time and the size, so it would be better to rename to `updateApplicationStoreInfo`.